horaedb/metric_engine/src/sst.rs

```diff
@@ -71,6 +74,18 @@ impl SstFile {
     pub fn mark_compaction(&self) {
         self.inner.in_compaction.store(true, Ordering::Relaxed);
     }
+
+    pub fn is_uncompacted(&self) -> bool {
+        !self.inner.in_compaction.load(Ordering::Relaxed)
+    }
```

Comment:
```suggestion
    pub fn is_compaction(&self) -> bool {
        self.inner.in_compaction.load(Ordering::Relaxed)
    }
```

Negate this bool inside this function is not very readable.